packages/calcite-ui-icons/docs/keywords.json

```diff
@@ -8970,5 +8970,50 @@
     "alias": ["3.32.0", "symbols", "check", "circle", "validated", "arrow", "complete"],
     "category": "Symbols",
     "release": "3.32.0"
+  },
+  "spaces": {
```

Comment:
Sidebar: would it be helpful to have all keywords organized alphabetically in the future?

Reply:
Not really since we don't have a need for them to be alphabetized

Reply:
I like this idea! It can be done automatically so it wouldn't add additional work for designers: https://www.npmjs.com/package/sort-json